packages/x-components/src/x-modules/search/store/index.ts

```diff
@@ -2,5 +2,6 @@ export * from './actions/fetch-and-save-search-response.action';
 export * from './actions/fetch-search-response.action';
```

Comment:
There is a big mess with x-module exports. There are actions or getters that are not exported, and some times the name even collides with the wires. I'm planning to fix this in another PR.